docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.8"
```

Comment:
@naufalmirk Should we create a separate docker-compose file here too? I assume it will need access to some things, like maybe the accounts service when we implement auth. I would lean to having one docker-compose file for now, even if that means you have to spin everything up.

Reply:
@naufalmirk Is this resolved?

Reply:
I'll add the rest of the compose file.

Reply:
@naufalmirk looks like I can't run both the frontend compose file and the api compose file together. Clashes over redis and errors out. So to me it seems like we either need:
1. 1 single compose file that includes api + frontend
2. or, a compose file for frontend + its specific dependencies, a compose file for API and its specific dependencies, and a common compose file for shared dependencies between API and Frontend.

My opinion is that option 1, although maybe less correct, is going to be less effort to maintain

Reply:
For anyone reading this, the outcome was:
* `dissolution-web/docker-compose.yml`  - Spins up everything (Frontend + API + dependencies). This is because API is a dependency of frontend, and there is no current use case why you would want to spin up frontend with the dissolution-api  stuff.
* `dissolution-api/docker-compose.yml`  - Spins up API and API dependencies, in case a developer would like to work on the API without the overhead of spinning up the frontend and the additional dependencies it brings with it i.e. ch.gov.uk.